src/main/java/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/StaticWhitelist.java

```diff
@@ -192,24 +194,31 @@ private static String printArgumentTypes(Class<?>[] parameterTypes) {
     }
 
     private static final Set<String> BLACKLIST;
-    static {
+
+    @SuppressFBWarnings(value = "OS_OPEN_STREAM", justification = "https://sourceforge.net/p/findbugs/bugs/786/")
```

Comment:
BTW when we can update the baseline to 1.625 or later, and thus set `java.level=7`, this can be replaced by the simpler

```java
try (InputStream is = StaticWhitelist.class.getResourceAsStream("blacklist");
     Reader r = new InputStreamReader(is, "US-ASCII");
     BufferedReader br = new BufferedReader(r)) {
    // …use br
}
```